lib/myfinance/resources/receivable_account.rb

```diff
@@ -26,7 +26,8 @@ def endpoints(id, entity_id, key)
         {
           create: '/entities/:entity_id/receivable_accounts',
           receive: '/entities/:entity_id/receivable_accounts/:id/receive',
-          undo_receivement: '/entities/:entity_id/receivable_accounts/:id/undo_receivement'
+          undo_receivement: '/entities/:entity_id/receivable_accounts/:id/undo_receivement',
+          destroy: '/entities/:entity_id/receivable_accounts/:id'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.